arrows/ceres/types.cxx

```diff
@@ -197,23 +168,6 @@ LossFunctionFactory(LossFunctionType type, double s)
   }
 }
 
-/// Return the number of distortion parameters required for each type
-unsigned int
-num_distortion_params(LensDistortionType type)
-{
-  switch(type)
-  {
-  case POLYNOMIAL_RADIAL_DISTORTION:
-    return distortion_poly_radial::num_coeffs;
-  case POLYNOMIAL_RADIAL_TANGENTIAL_DISTORTION:
-    return distortion_poly_radial_tangential::num_coeffs;
-  case RATIONAL_RADIAL_TANGENTIAL_DISTORTION:
-    return distortion_ratpoly_radial_tangential::num_coeffs;
-  default:
-    return 0;
-  }
-}
-
 } // end namespace ceres
 } // end namespace arrows
 } // end namespace kwiver
```

Comment:
Looks like this didn't get uncrustified?

Reply:
uncrustified